bundles/org.openhab.core.transform/src/test/java/org/openhab/core/transform/internal/TransformationConfigurationRegistryImplTest.java

```diff
@@ -47,22 +47,22 @@ public class TransformationConfigurationRegistryImplTest {
     private static final String MANAGED_WITH_DE_LANGUAGE_UID = "config:" + SERVICE + ":managed:de";
 
     private static final TransformationConfiguration MANAGED_WITHOUT_LANGUAGE = new TransformationConfiguration(
-            MANAGED_WITHOUT_LANGUAGE_UID, "", SERVICE, null, MANAGED_WITHOUT_LANGUAGE_UID);
+            MANAGED_WITHOUT_LANGUAGE_UID, "", SERVICE, SERVICE, null, MANAGED_WITHOUT_LANGUAGE_UID);
```

Comment:
This shows what I mention above: As an API user, one does not expect to pass JSON and instead passes some other stuff, like duplicated and thus useless information like here.

Reply:
That's just a leftover from my first attempt (a "subType" which contained either additional information or the duplicated information). But this turned out not to be sufficient for blocky.